README.md

```diff
@@ -0,0 +1,174 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+  <h1><b>Budget-app README</b></h3>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [🎥 Video Demonstration](#video-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget-App <a name="about-project"></a>
+
+_Budget-app_ is a mobile web application designed for efficient budget management. Users can register, log in, add transactions to categories, and monitor their expenses. With secure authentication, PostgreSQL database, and comprehensive testing, this app ensures seamless and secure budget tracking.
+
+<img src="./app/assets/images/schema.png" alt="UML class diagram"/>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/documentation/">Ruby</a></li>
+    <li><a href="https://rspec.info//">Rspec</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- _1 - Create Category_
+- _2 - Create Transaction_
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budgetapp-mu8y.onrender.com/)
+
+- [Presentation Loom Video](https://www.loom.com/share/1b4f1ef209e94528bf73a1cf2ccb182c?sid=d3588348-3e8c-48ba-8ccf-3057649bc7af)
+
```

Comment:
- [ ] While you did well updating the demo video, I was having a challenge hearing what you were saying having in some sections being in audible, Kindly do the following:
   -  KIndly retake the video ensuring that your microphone is clear and that people can hear you clear;y.
   -  KIndly record a video demonstrating the local version of the project and not the deployed version.


Once this is done, I believe your project will be ready for approval. :rocket:

Reply:
Hello @Nemwel-Boniface 

Thanks for your detailed comments and guidance on what to improve. As per your request, I have redone the loom video.

Plz check and let me know.